Gemfile

```diff
@@ -6,4 +6,5 @@ gem 'jruby-openssl', :platforms => :jruby
 group :test, :remote_test do
   # gateway-specific dependencies, keeping these gems out of the gemspec
   gem 'braintree', '>= 2.0.0'
+  # gem 'braintree', github: "braintree/braintree_ruby", branch: "raw_apple_pay_preview" # for apple pay support in braintree
```

Comment:
If the braintree gem is included so that the braintree tests pass - it seems like if that's what you need for those tests to pass, that should be what's in the gemfile (and then this can get updated to go back to the regular gem once it's been released).